package.json

```diff
@@ -18,8 +18,7 @@
     "Linters"
   ],
   "activationEvents": [
-    "onLanguage:crystal",
-    "onCommand:crystal.ameba.lint"
```

Comment:
Why was that event removed?

Reply:
<img width="573" alt="image" src="https://github.com/user-attachments/assets/ae683a80-dc4b-47ab-9266-600742c8b586">

Reply:
I see, in such case we also need to bump the supported vscode version to `^1.75`.